### PR TITLE
Add Exercise class and attribute classes implementation

### LINKED ISSUE #78: Add exercise command (add-e)
TODOs:

1. Storage @AaronCQL 
2. Model @Yonggiee

### LINKED ISSUE #80: As a coach, I can record the exercise type and intensity my clients have done for the day

### PR DESCRIPTION
**Pull request by Yonggiee**
- tests implementation not done

### CODE REVIEW

**Review by AaronCQL — COMMENTED** [review]
Perhaps we should be more consistent with the file namings under the `seedu.address.model.exercise` package. For the sake of consistency with the other files in this package, should we rename `Sets` and `Reps` to `ExerciseSets` and `ExerciseReps` instead?

**Review by AaronCQL — APPROVED** [review]
LGTM

File: src/main/java/seedu/address/logic/commands/AddExerciseCommand.java [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+/**
+ * Adds an exercise done by a client in FitBiz.
+ */
+public class AddExerciseCommand extends Command {
+
+    public static final String COMMAND_WORD = "add-e";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds an exercise done by a client in FitBiz. "
+        + "Parameters: INDEX (must be a positive integer) "
```
Comment (ziyingli) [inline src/main/java/seedu/address/logic/commands/AddExerciseCommand.java]: shld not have le

File: src/main/java/seedu/address/logic/commands/AddExerciseCommand.java [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+        + PREFIX_NAME + "EXERCISE_NAME "
+        + PREFIX_DATE + "DATE "
+        + "[" + PREFIX_REPS + "REPS] "
+        + "[" + PREFIX_EXERCISE_WEIGHT + "EXERCISE_WEIGHT] "
+        + "[" + PREFIX_SETS + "SETS]\n"
+        + "Example: " + COMMAND_WORD + " 1 "
```
Comment (ziyingli) [inline src/main/java/seedu/address/logic/commands/AddExerciseCommand.java]: the `+ " 1 "` also

File: src/main/java/seedu/address/logic/commands/AddExerciseCommand.java (hunk 1 above)
Comment (Yonggiee) [reply src/main/java/seedu/address/logic/commands/AddExerciseCommand.java]: done

File: src/main/java/seedu/address/logic/commands/AddExerciseCommand.java (hunk 2 above)
Comment (Yonggiee) [reply src/main/java/seedu/address/logic/commands/AddExerciseCommand.java]: done

File: src/main/java/seedu/address/logic/commands/AddExerciseCommand.java [hunk 3]
```diff
@@ -24,13 +24,12 @@
     public static final String COMMAND_WORD = "add-e";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds an exercise done by a client in FitBiz. "
-        + "Parameters: INDEX (must be a positive integer) "
```
Comment (AaronCQL) [inline src/main/java/seedu/address/logic/commands/AddExerciseCommand.java]: Why is the `"Parameters: "` also removed?

File: src/main/java/seedu/address/logic/commands/AddExerciseCommand.java [hunk 4]
```diff
@@ -24,13 +24,12 @@
...
         + PREFIX_NAME + "EXERCISE_NAME "
         + PREFIX_DATE + "DATE "
         + "[" + PREFIX_REPS + "REPS] "
         + "[" + PREFIX_EXERCISE_WEIGHT + "EXERCISE_WEIGHT] "
         + "[" + PREFIX_SETS + "SETS]\n"
-        + "Example: " + COMMAND_WORD + " 1 "
+        + "Example: "
```
Comment (AaronCQL) [inline src/main/java/seedu/address/logic/commands/AddExerciseCommand.java]: Why did you remove the `COMMAND_WORD`?

**Review by AaronCQL — CHANGES_REQUESTED** [review]
Wanky

File: src/main/java/seedu/address/logic/commands/AddExerciseCommand.java (hunk 3 above)
Comment (Yonggiee) [reply src/main/java/seedu/address/logic/commands/AddExerciseCommand.java]: oof fkup

**Review by AaronCQL — APPROVED** [review]
